### ISSUE TITLE
Big Merge with the upstream SWC hpc-intro repo

### ISSUE THREAD

**Issue by cdkharris** [UCL-ARC/hpc-intro#31]
### Definition of Done / Acceptance Criteria

DoD: Our repo is no longer far behind the upstream repo.

Our most recent common ancestor with the upstream repo is June 2022. Merging will let us benefit from the work they've been doing over the last two years and will let us look at the differences between our lesson and the upstream so we can find pieces we should contribute back.

Some major benefits we're merging in:

- A new lesson on Environment Variables
- An overhauled lesson on Parallel Computing
- New sites

### Testing

Use `make serve` locally to check that the site functions and review how the site is presented, then manually compare it with the [upstream](https://carpentries-incubator.github.io/hpc-intro/) and the currently-published [ARC site](https://github-pages.arc.ucl.ac.uk/hpc-intro/).

### Dependencies

These changes should not be pulled into the main branch until after this term's course has run.

### Comments

Working on this here: https://github.com/UCL-ARC/hpc-intro/tree/cdkharris/big-merge

After the big merge, we still need to review the lessons before we can contribute upstream or use the merged lesson to run a course. In particular, we need to review these three lessons:

- Environment Variables, as it's a totally new lesson.
- Modules, as we have updated the version of modules we run this year.
- Parallel, as it's been significantly overhauled.

**Comment by NicoleLabrAvila** [comment]
Hi Camilla!
I found an error in one of the examples in the scheduler section: 
https://github-pages.arc.ucl.ac.uk/hpc-intro/13-scheduler/index.html

This part :

#!/bin/bash -l
#$ -N long_job
#$ -l h_rt= 00:01 # timeout in HH:MM

echo "This script is running on ... "
sleep 240 # time in seconds
hostname

It did not run  in Myriad and sent errors. For me only worked if I changed this line : 
#$ -l h_rt= 00:01 # timeout in HH:MM

for this:
#$ -l h_rt=00:01:00 # timeout in HH:MM

There was one space extra and the time needed to have the "seconds" part. 
Hope this helps!
Bests,

**Comment by cdkharris** [comment]
Hi Nicole, thanks; I've made this change on the branch: https://github.com/UCL-ARC/hpc-intro/commit/aee8d050c8cb9115c45c38710b3acd58f8bd2e92

Please keep a list of these bugs in the lessons, perhaps on a new issue; after we do this big merge we'll need to go through all the lessons and check that things are working. But because the big merge will change a lot of the lessons I think it will be easier to wait until after we merge to make a new branch and actually make the changes to update the lessons. And I can't finish merging this branch until after we've run the course for this year, since it would change the website.

Maybe we should have a dev branch for this repository where we can stage changes before merging them onto the main gh-pages branch.

**Comment by NicoleLabrAvila** [comment]
Oki, I will keep a record! With respect to the branches, a dev branch sounds like a good idea in case I or other people find more issues

**Comment by cdkharris** [comment]
I finished doing the big merge, a lot of it by hand, but the saga continues in #34
